tavern/tomes/ifconfig/main.eldritch

```diff
@@ -1,8 +1,26 @@
+def print_table(rows: list[list]):
+    """Pretty print a table, auto adjusting width based on the row data"""
+    def rpad(s: str, n: int, c=" "):
+        return s + (n-len(s))*c
+    # count columns
+    counts = [0]*len(rows[0])
+    for r in rows:
+        for i, f in enumerate(r):
+            counts[i] = max(len(f), counts[i])
+    # Print the columns
+    for r in rows:
+        row = []
+        for i, c in enumerate(r):
+            row.append(rpad(c, counts[i]+2))
+        print("".join(row))
+
 def ifconfig():
```

Comment:
Let's rename the tome from ifconfig to something less loaded like `get_net_info` or `network_recon`
Then in the future if we have other net info like trace route we can include it here as well.